sonar-project.properties

```diff
@@ -1,3 +1,6 @@
 sonar.projectKey=dpm
 sonar.sources=./src
 sonar.qualitygate.wait=true
+
+# Exclude development files from analysis
```

Comment:
Excluded this file since is a screen used for development puropose only and the static analysis is reporting ~62% of duplicated code in this file.